artifact.go

```diff
@@ -108,7 +108,7 @@ func NewArtifact(img types.Image, log logrus.FieldLogger, c CacheClient, opt art
 		log:            log,
 		image:          img,
 		cache:          c,
-		walker:         walker.NewLayerTar(opt.SkipFiles, opt.SkipDirs, opt.Slow),
+		walker:         walker.NewLayerTar(opt.SkipFiles, opt.SkipDirs),
```

Comment:
Add test for artifact and Inspect method similar as we have for full collect https://github.com/castai/kvisor/blob/main/cmd/agent/imagescan/collector/collector_test.go#L30